apps/admin/app/(auth)/register/_components/form.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import { AuthPage, AuthProviderButton } from "@repo/ui/components";
+import { formSchema, TFormSchema } from "./form-schema";
+import { useForm } from "react-hook-form";
+import { ReactForm } from "@repo/ui/forms";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Button, Input, Linktag, Textbox } from "@repo/ui/partials";
+import Link from "next/link";
+
+
+
+export function RegisterationForm() {
+  const form = useForm<TFormSchema>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+      password: "",
+      firstName: "",
+      lastName: "",
+    },
+  });
+
+  const submitfunction = () => {
+    console.log("Submitted");
```

Comment:
```suggestion
    console.log("Submitted");
```
please remove console

Reply:
@yaju09 This will be definitely changed when functionality is implemented. This is just a test function